pkg/kmsca/kmsca.go

```diff
@@ -80,7 +80,7 @@ func (ca *KMSCA) CreateKey(input *CreateKeyInput) (string, error) {
 		CustomerMasterKeySpec: aws.String(kms.CustomerMasterKeySpecRsa2048),
 	}
 	if len(input.CustomerMasterKeySpec) > 0 {
-		keyInput.CustomerMasterKeySpec = aws.String(input.CustomerMasterKeySpec)
+		keyInput.KeySpec = aws.String(input.CustomerMasterKeySpec)
```

Comment:
`CustomerMasterKeySpec` is depricated but at the moment we still set it on line 80. We should change it there too.

Reply:
Done